src/code/z_en_item00.c

```diff
@@ -71,66 +71,20 @@ static void* sItemDropTex[] = {
     gDropMagicLargeTex,    gDropMagicSmallTex, gDropDekuSeedsTex, gDropKeySmallTex,
 };
 
+#define DROP_TABLE_ROW(d0, d1, d2, d3, d4, d5, d6, d7, d8, d9, dA, dB, dC, dD, dE, dF) \
+    (d0), (d1), (d2), (d3), (d4), (d5), (d6), (d7), (d8), (d9), (dA), (dB), (dC), (dD), (dE), (dF),
+#define DROP(item, quantity) (ITEM00_##item)
+
```

Comment:
The string concatenation is probably not desirable cause it hinders searchability. And also since there are a million different enums with these item names in them, id say its better to have it more clear in the macro definitions that these are the ITEM00_ enum values